src/main/java/org/hiero/sketch/table/Quantiles.java

```diff
@@ -0,0 +1,35 @@
+package org.hiero.sketch.table;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Random;
+
+public class Quantiles {
+    private final int inpSize;
+    private final int [] inpArray;
+    private final Comparator<Integer> comp;
+
+    public Quantiles(final int [] input, final Comparator<Integer> comp){
```

Comment:
You are missing several spaces )_{
Is this Comparator<Integer> or RowComparator?